### ISSUE TITLE
Constant properties whose value is not a single number cause the whole class to be not documented

### ISSUE THREAD

**Issue by simgunz** [simgunz/doxymatlab#1]
Example code:

```
properties (Constant)
    %> Supported methods
    supportedMethods = {'A', 'B'};
end
```

Only the file documentation is produced, but not the class documentation.

**Comment by bonanza123** [comment]
I just tried it, meaning I added your code above into the `classDocumentationExample.m`, removed the old documentation folder and recompiled. The documentation file `html/classclass_documentation_example.html` contains everything (also the `supportedMethods` cell).

Did I misunderstand anything or may it have been a doxygen rather than doxymatlab issue?

**Comment by simgunz** [comment]
Actually the problem was related to the naming of the variables. When they contain - the issue is triggered. 
So also the example is wrong.
The naming problme is fixed by Fixed by 0a925f370453f4f263752020298384c027596ebc
